pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -70,6 +72,16 @@ func Add(mgr manager.Manager, configInfo *types.ConfigInfo, volumeManager volume
 	defer cancel()
 	ctx = logger.NewContextWithLogger(ctx)
 	log := logger.GetLogger(ctx)
+	// Initialize the k8s orchestrator interface
+	coCommonInterface, err := commonco.GetContainerOrchestratorInterface(ctx, common.Kubernetes, cnstypes.CnsClusterFlavorWorkload, &syncer.COInitParams)
+	if err != nil {
+		log.Errorf("failed to create CO agnostic interface. Err: %v", err)
+		return err
+	}
+	if !coCommonInterface.IsFSSEnabled(ctx, common.FileVolume) {
+		log.Infof("File volume feature is disabled on the cluster")
```

Comment:
log.Infof("File volume feature is disabled on the cluster") -> log.Infof("Not initializing the CnsFileAccessConfig Controller as File volume feature is disabled on the cluster")

Reply:
Addressed in the latest commit and squashed. PTAL